cmd/cluster-version-operator-tests/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-eng/openshift-tests-extension/pkg/cmd"
+	"github.com/openshift-eng/openshift-tests-extension/pkg/extension"
+	"github.com/openshift-eng/openshift-tests-extension/pkg/extension/extensiontests"
+)
+
+func main() {
+	registry := extension.NewRegistry()
+	ext := extension.NewExtension("openshift", "payload", "cluster-version-operator")
+
+	var specs []*extensiontests.ExtensionTestSpec
```

Comment:
Maybe https://github.com/openshift/cluster-version-operator/blob/40820712f1119f92f97fa473e232beb3829953c2/vendor/github.com/openshift-eng/openshift-tests-extension/pkg/extension/extension.go#L49?

```suggestion
	var specs extensiontests.ExtensionTestSpecs
```

It is also used in the enhancement.

Reply:
That's better, thanks. Done.